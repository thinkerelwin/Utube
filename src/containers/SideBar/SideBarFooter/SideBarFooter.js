import React from 'react';
import './SideBarFooter.scss';

export function SideBarFooter() {
  return (
    <>
      <div className="footer-block">
        <div>
          <span>About</span> <span>Press</span> <span>Copyright</span>
        </div>
        <div>
          <span>Contact Us</span>
          <span>Creators</span>
        </div>
        <div>
          <span>Advertise</span>
          <span>Developers</span>
        </div>
      </div>
      <div className="footer-block">
        <div>
          <span>Terms</span> <span>Privacy</span>
          <span>Policy & Safety</span>
        </div>
        <div>Test new features</div>
      </div>
      <div className="footer-block">
        <div>Â© Utube.com - A Youtube clone for learing purposes .</div>
      </div>
    </>
  );
}
