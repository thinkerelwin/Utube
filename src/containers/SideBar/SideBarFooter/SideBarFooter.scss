.footer-block {
  padding-bottom: 10px;
  padding-left: 16px;
  color: #918888;
  span {
    margin-right: 8px;
  }
}
