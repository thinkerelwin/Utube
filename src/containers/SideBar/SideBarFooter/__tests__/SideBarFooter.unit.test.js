import React from 'react';
import { shallow } from 'enzyme';
import { SideBarFooter } from '../SideBarFooter';

test('renders SideBarFooter', () => {
  const wrapper = shallow(<SideBarFooter />);
  expect(wrapper).toMatchSnapshot();
});
