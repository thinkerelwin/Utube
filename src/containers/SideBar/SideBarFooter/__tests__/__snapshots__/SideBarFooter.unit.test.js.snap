// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SideBarFooter 1`] = `
<Fragment>
  <div
    className="footer-block"
  >
    <div>
      <span>
        About
      </span>
       
      <span>
        Press
      </span>
       
      <span>
        Copyright
      </span>
    </div>
    <div>
      <span>
        Contact Us
      </span>
      <span>
        Creators
      </span>
    </div>
    <div>
      <span>
        Advertise
      </span>
      <span>
        Developers
      </span>
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      <span>
        Terms
      </span>
       
      <span>
        Privacy
      </span>
      <span>
        Policy & Safety
      </span>
    </div>
    <div>
      Test new features
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Â© Utube.com - A Youtube clone for learing purposes .
    </div>
  </div>
</Fragment>
`;
